airbyte-protocol/models/src/main/resources/protocol_models/airbyte_message.yaml

```diff
@@ -0,0 +1,78 @@
+---
+"$schema": http://json-schema.org/draft-07/schema#
+"$id": https://github.com/airbytehq/airbyte/blob/master/airbyte-protocol/models/src/main/resources/protocol_models/airbyte_message.yaml
+title: AirbyteMessage
+description:
+type: object
+additionalProperties: true
```

Comment:
why do we want to allow additional properties?

Reply:
bad copy-paste